src/client/utils/browser.js

```diff
@@ -1,48 +1,50 @@
-function getMSEdgeVersion (userAgent) {
-    var edgeStrIndex = userAgent.indexOf('edge/');
-
-    return parseInt(userAgent.substring(edgeStrIndex + 5, userAgent.indexOf('.', edgeStrIndex)), 10);
-}
-
 function calculateBrowserAndVersion (userAgent) {
-    var webkitRegEx  = /(webkit)[ \/]([\w.]+)/;
-    var msieRegEx    = /(msie) ([\w.]+)/;
-    var firefoxRegEx = /(firefox)/;
-
-    var match = webkitRegEx.exec(userAgent) ||
-                msieRegEx.exec(userAgent) ||
-                userAgent.indexOf('compatible') < 0 && firefoxRegEx.exec(userAgent) ||
-                [];
+    var msEdgeRegEx   = /(edge)\/([\w.]+)/;
```

Comment:
We use another naming convention for inplace regexps:
https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/sandbox/event/simulator.js#L303
https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/utils/dom.js#L29